### PR TITLE
Add: wmutils-core

### PR DESCRIPTION
**Pull request by sxv**
my first pull-request, please review carefully :)

### CODE REVIEW

File: pkgs/tools/X11/wmutils-core/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/wmutils/core/archive/v1.0.tar.gz";
+    sha256 = "10vn56rbrjykcrjr06ki4qc12sri1ywrcvm89nmxlqhkxx4i239p";
+  };
+
+  buildInputs = [ libxcb ];
+
+  buildPhase = ''
```
Comment (lethalman) [inline pkgs/tools/X11/wmutils-core/default.nix]: This is not needed.

File: pkgs/tools/X11/wmutils-core/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    make PREFIX=$out
+  '';
+
+  installPhase = ''
```
Comment (lethalman) [inline pkgs/tools/X11/wmutils-core/default.nix]: This is not needed.

File: pkgs/tools/X11/wmutils-core/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchurl, libxcb }:
+
+stdenv.mkDerivation rec {
+  name = "wmutils-core-${version}";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://github.com/wmutils/core/archive/v1.0.tar.gz";
```
Comment (lethalman) [inline pkgs/tools/X11/wmutils-core/default.nix]: Use `${version}` so that you update the version only in one place.

**Comment by lethalman** [comment]
The derivation is fine, I just pointed you a couple of things for your own knowledge.

**Comment by sxv** [comment]
Thank you, that is helpful. Can you elaborate on the "not needed"? the buildPhase and installPhase blocks?

File: pkgs/tools/X11/wmutils-core/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  buildInputs = [ libxcb ];
+
+  makeFlags = [ "PREFIX=$out" ];
+  
+  installFlags = [ "PREFIX=$out" ];
+
+  meta = with lib; {
+    description = "Set of window manipulation tools";
+    homepage = https://github.com/wmutils/core;
+	license = stdenv.lib.licenses.isc;
```
Comment (lethalman) [inline pkgs/tools/X11/wmutils-core/default.nix]: Fix indentation here, please. Must be 2 spaces.

**Comment by lethalman** [comment]
Uhm interesting, the resulting dir is empty. Hope you tried this :) It should be `installFlags = [ "PREFIX=$(out)" ]`. Also I think that the PREFIX does little when doing make.

**Comment by vcunat** [comment]
After using the @lethalman's suggestion, the output looks fine. Squashed, refactored a little, and pushed.

**Comment by vcunat** [comment]
Ah, I forgot to hand over the authorship of the squash before pushing, I'm sorry (silly ``git merge --squash`` default).

**Pull request closed without merging**